packages/in_app_purchase/in_app_purchase_platform_interface/lib/src/in_app_purchase_platform.dart

```diff
@@ -24,15 +24,16 @@ abstract class InAppPurchasePlatform extends PlatformInterface {
   /// The instance of [InAppPurchasePlatform] to use.
   ///
   /// Defaults to `null`.
-  static InAppPurchasePlatform? get instance => _instance;
+  static InAppPurchasePlatform get instance => _instance;
 
-  static InAppPurchasePlatform? _instance;
+  // Should only be accessed after setter is called.
+  static late InAppPurchasePlatform _instance;
```

Comment:
This should be either above or below the getter and setter, rather than between them.

Reply:
done